pyproject.toml

```
@@ -12,14 +12,15 @@ license = "Apache 2.0"
 [tool.poetry.dependencies]
 python = "<3.11,>=3.7.1"
 requests = "^2.25.1"
-singer-sdk = "^0.5.0"
+singer-sdk = "0.13.0"
```

```suggestion
singer-sdk = "^0.13.0"
```
we probably don't want to pin to such a specific version, especially for a 0.x. Docs on what the `^` does [here](https://python-poetry.org/docs/dependency-specification/#caret-requirements)